src/USER-MDI/fix_mdi_engine.h

```diff
@@ -0,0 +1,137 @@
+/* -*- c++ -*- ----------------------------------------------------------
+   LAMMPS - Large-scale Atomic/Molecular Massively Parallel Simulator
+   http://lammps.sandia.gov, Sandia National Laboratories
+   Steve Plimpton, sjplimp@sandia.gov
+
+   Copyright (2003) Sandia Corporation.  Under the terms of Contract
+   DE-AC04-94AL85000 with Sandia Corporation, the U.S. Government retains
+   certain rights in this software.  This software is distributed under
+   the GNU General Public License.
+
+   See the README file in the top-level LAMMPS directory.
+------------------------------------------------------------------------- */
+
+
+#ifdef FIX_CLASS
+
+FixStyle(mdi/engine,FixMDIEngine)
+
+#else
+
+#ifndef LMP_FIX_MDI_H
```

Comment:
this include guard should be called `LMP_FIX_MDI_ENGINE_H`

Reply:
Fixed in b360be4.